lib/elixir/src/elixir.erl

```diff
@@ -348,6 +348,7 @@ parser_line(Meta) ->
   end.
 
 to_binary(List) when is_list(List) -> elixir_utils:characters_to_binary(List);
+to_binary(Binary) when is_binary(Binary) -> Binary;
```

Comment:
Why is this change necessary?

Reply:
oh I was getting an error since we are passing a binary from the error message [here](https://github.com/elixir-lang/elixir/pull/7809/files#diff-8e79b6aba4faa9923e1ed2defe97e290R614).

Reply:
I see. It is the same "principle" as before then. We should not introduce new "formats" for returning errors. We already have a well defined tuple, we should stick to it. :)